### ISSUE TITLE
A11y: Additionally displayed content cannot be closed without moving the focus (e.g. by pressing the Escape key)

### ISSUE THREAD

**Issue by ChristofSimon** [grafana/grafana#82152]
**Steps to reproduce**:
display a x-y-diagram and move the mouse cursor (or keyboard cursor) within the area of the diagram so that additional information (measures and values) are displayed, then try co close that displayed content without moving the focus (e.g. by pressing the Escape key)

**Actual Result**:
the content does not close
![1 4 13_diagramm-close-content](https://github.com/grafana/grafana/assets/122528177/e5c858aa-8f88-4b40-aa30-9a8bbda6a2ba)

**Expected Result**
content cant be closed without moving the focus (e.g. by pressing the Escape key)

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
1.4.13 Content on Hover or Focus: https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus

**Environment**:
- Grafana version: 9.1.0
- Data source type & version:
- User OS & Browser: Windows 10, Chrome Version 107.0.5304.88
- Others:

**Comment by tonypowa** [comment]
hi @ChristofSimon 

thank you for reporting this issue

Is this A11y issue specific to a particular type of panel, or is it consistent across panels?

**Comment by ChristofSimon** [comment]
Hi @tonypowa 
I'll try to find out the answer and get back.

**Comment by ChristofSimon** [comment]
Sorry for the delay. 
This issue is not specific to a particular type of panel. It is consistent across many panels and it also persists in Grafana version 11.2.0.
Here are two examples:

![Image](https://github.com/user-attachments/assets/fcc74521-e206-4e92-9ba6-2ff5926d8918)

![Image](https://github.com/user-attachments/assets/03a3631f-2e89-41df-afcb-01a3793dd13a)
